lib/devise-guests/controllers/helpers.rb

```diff
@@ -71,7 +71,7 @@ def create_guest_#{mapping} key = nil
           end
         end
 
-        def guest_email_authentication_key key
+        def guest_#{class_name.constantize.authentication_keys.first}_authentication_key key
```

Comment:
This doesn't seem right. With the suggested change you would just be populating your `phone` attribute with a guest email address.

A better solution might be to update the README with instructions for adding your own `guest_phone_authentication_key` method which will return the right guest-mocked content?